code/menuui/readyroom.cpp

```diff
@@ -1199,7 +1199,7 @@ void sim_room_close()
 //
 void sim_room_do_frame(float  /*frametime*/)
 {
-	char buf[256];
+	char buf[CF_MAX_PATHNAME_LENGTH + 4];
```

Comment:
I actually seem to have changed the wrong size here - it's the full_name a few lines later that needs to hold a path+extension. (Increasing that size is necessary there because Campaign.filename is potentially longer)